src/main/java/com/alevel/module2/models/Invoice.java

```diff
@@ -0,0 +1,116 @@
+package com.alevel.module2.models;
+
+import lombok.Getter;
+
+import java.util.Date;
+
+@Getter
+public class Invoice {
+    private static int limit;
+
+    private final Date date;
+    private final Technics[] technic;
+    private final Customer customer;
+    private TypeSale type;
+
+    public Invoice(Technics[] technic, Customer customer, Date date) {
+        this.technic = technic;
+        this.customer = customer;
+        this.date = date;
+    }
+
+    public int countTelephone() {
+        int count = 0;
+        for (Technics technics : technic) {
+            if (technics.getType().equals("Telephone")) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public int countTelevision() {
+        int count = 0;
+        for (Technics technics : technic) {
+            if (technics.getType().equals("Television")) {
+                count++;
+            }
+        }
+        return count;
+    }
```

Comment:
```suggestion
 public long countDeviceByType(String type) {
        return Arrays.stream(technic)
                .filter(technics -> technics.getType().equals(type))
                .count();
    }
```

But the Invoice should not do it